From 7dec24e81c8192495f920680220ccb8d467280dd Mon Sep 17 00:00:00 2001
From: Srikanth Krishnakar <Srikanth_Krishnakar@mentor.com>
Date: Thu, 10 Dec 2015 21:53:27 +0530
Subject: [PATCH 5/5] kernel/module.c: Remove optimization for
 complete_formation

We need this to retain local variable in the context.

Signed-off-by: Srikanth Krishnakar <Srikanth_Krishnakar@mentor.com>
---
 kernel/module.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel/module.c b/kernel/module.c
index bb8cef4..aa403d8 100644
--- a/kernel/module.c
+++ b/kernel/module.c
@@ -3147,6 +3147,7 @@ out_unlocked:
 	return err;
 }
 
+__attribute__((optimize(0)))
 static int complete_formation(struct module *mod, struct load_info *info)
 {
 	int err;
-- 
1.9.1

